test/pom.xml

```diff
@@ -24,7 +24,7 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>${junit.version}</version>
+            <scope>compile</scope>
```

Comment:
it is not the default since i moved the dep up to dependency manager in a parent pom. this needs to be reset to compile since test package is making a deployable for others to load